### PR TITLE
porting frost-button

### PR DESCRIPTION
**Pull request by vesper2000**
# patch

straight port from ember-frost-button

### CODE REVIEW

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/14/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: tests/dummy/app/pods/button/template.hbs [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+}}
+  Vertical
+{{/frost-checkbox}}
+
+<br/>
+
+<div class="section">
+  <div class="example">
+    <div class="title">Primary</div>
+    <div class="code">
```
Comment (rox163) [inline tests/dummy/app/pods/button/template.hbs:20]: will need a change to demo on top then code snippet below to be consistent with others

File: tests/dummy/app/pods/button/template.hbs [hunk 2]
```diff
@@ -0,0 +1,387 @@
+<div class="dummy-notification-container">
+  {{#each notifications as |notification|}}
+      {{notification-message notification=notification}}
+  {{/each}}
+</div>
+
+<h2>frost-button</h2>
```
Comment (rox163) [inline tests/dummy/app/pods/button/template.hbs:7]: remove this heading

File: tests/dummy/app/pods/button/template.hbs (hunk 1 above)
Comment (vesper2000) [inline tests/dummy/app/pods/button/template.hbs:20]: This is just a port - no updates to make things consistent.

**Pull request merged**